packages/flx/lib/signing.dart

```diff
@@ -165,8 +165,8 @@ ECSignature _asn1ParseSignature(Uint8List signature) {
 }
 
 PublicKey _publicKeyFromPrivateKey(ECPrivateKey privateKey) {
-  ECPoint Q = privateKey.parameters.G * privateKey.d;
-  return new ECPublicKey(Q, privateKey.parameters);
+  ECPoint q = privateKey.parameters.G * privateKey.d;
```

Comment:
I'm on `1.16.0-dev.3.0` (from yesterday) and am not seeing the lint reported for this case, so presumably it just landed :)